lib/services/batchaiManagement/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "azure-batchai",
+  "author": "Microsoft Corporation",
+  "contributors": [
+    "Alexander, Yukhanov <alyukha@microsoft.com>"
+  ],
+  "version": "1.0.0",
+  "description": "Microsoft Azure Batch AI Resource Provider Management Client Library for node",
+  "tags": [
+    "azure",
+    "sdk"
+  ],
+  "keywords": [
+    "node",
+    "azure"
+  ],
+  "main": "./lib/batch.js",
+  "types": "./lib/batch.d.ts",
+  "license": "MIT",
+  "dependencies": {
+    "ms-rest": "^2.2.2",
+    "ms-rest-azure": "^2.3.3",
+    "underscore": "^1.8.3",
+    "moment": "~2.18.1",
+    "@types/underscore": "^1.8.0"
```

Comment:
why do you need `"@types/underscore"` in dependencies?

Reply:
removed